muxpi/set-up-image.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -e
+
+#If you want to run the neptune image, run `./set-up-image.sh neptune`
+NEPTUNE_IMAGE="$1"
+COUNTER=5
+
+HOME="/home/muxpi"
+IMAGES="${HOME}/images"
+
+#The url for the latest successful nightly build.
+URL_MINIMAL="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY/lastSuccessfulBuild/artifact/artifacts_rpi/core-image-pelux-minimal-dev-raspberrypi3.rpi-sdimg"
+VARIANT_MINIMAL="core-image-pelux-minimal-dev-raspberrypi3.rpi-sdimg"
+
+URL_NEPTUNE="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY/lastSuccessfulBuild/artifact/artifacts_rpi-qtauto/core-image-pelux-qtauto-neptune-dev-raspberrypi3.rpi-sdimg"
+VARIANT_NEPTUNE="core-image-pelux-qtauto-neptune-dev-raspberrypi3.rpi-sdimg"
+
+mkdir -p /home/muxpi/images
+
+if [ "$NEPTUNE_IMAGE" = "neptune" ]; then
+   wget $URL_NEPTUNE -P $IMAGES
+   if [ $? == 0 ]; then
+      echo "Image downloaded"
+   fi
+   echo "{\"${VARIANT_NEPTUNE}\":\"\"}" > $HOME/map.json
+   echo "Json map is ready. Compressing the downloaded image... "
+   
+   cd $IMAGES
+   tar -czvf $VARIANT_NEPTUNE.tar.gz $VARIANT_NEPTUNE
+   if [ $? == 0 ]; then
+      echo "Image has finished compressing" 
+   fi
+   # fota requires the card device to be flashed, the json map which contains
+   # the image and partitions, and finally the compressed image on tar.gz
+   fota_armv7 -card /dev/sda -map $HOME/map.json $IMAGES/$VARIANT_NEPTUNE.tar.gz 
+   if [ $? == 0 ];then
+      echo "Neptune image successfully flashed using fota"
+   fi
+fi
+
+if [ "$NEPTUNE_IMAGE" != "neptune" ]; then
```

Comment:
You should use an if/else for setting the image to use. This is unnecessary duplicate code. Even the printout at the end could be defined by if the neptune flag is set or not.